packages/react-charting/docs/TestPlans/DonutChart/UnitTests.md

```diff
@@ -0,0 +1,23 @@
+# Unit test plan for Donut Chart
+
+This test plan contains the list of unit testable functions which are used as a part of the Donut Chart component.
+
+1.  Identify the functions that can be unit tested (example, functions having calculations or getting values from Utils, etc).
+    If required, extract the unit testable portions out of the functions which can be independently unit tested without any requirement of DOM elements.
```

Comment:
Add statement to alternatively mock the section that cannot be unit tested.

Reply:
done